src/components/patterns/ski-scotland/ski-scotland-status/SkiScotlandStatus.vue

```diff
@@ -930,8 +932,28 @@ export default {
                 output.lastUpdate = data.lastupdate;
                 [output.lifts] = data.lift.sectors;
                 output.lifts = output.lifts.lifts;
-                [output.runs] = data.run.areas;
-                output.runs = output.runs.runs;
+                const areaRuns = data.run.areas.map((area) => area.runs);
+
+                // Some of the sites (Nevis Range) return multiple areas, some runs appear in
+                // multiple areas and some are only in one so we have to join them, then filter
+                // out dupes.
+                let runs = [];
+                for (let x = 0; x < areaRuns.length; x++) {
```

Comment:
Just an FYI - This could also be done using map - a nice article on map over for loops here https://medium.com/@ExplosionPills/map-vs-for-loop-2b4ce659fb03 
You can chain map and filter and makes it quite neat.

Reply:
Aah, yea I had the previous stage as a map but couldn't figure out how to get map to return a single concated array. I was missing .reduce which is much neater here to get from the map to the filter